CenterForOpenScience/ember-osf-web
# [ENG-4421] No BsForm

-   Ticket: [ENG-4421]
-   Feature flag: n/a

## Purpose

Remove all the `BsForm`s, which also involves removing all the `ValidatedModelForm`s, which are based on `BsForm`. Either just get rid of them or replace them with `FormControls`

## Summary of Changes

1. Remove unused settings.profile.name form
2. Remove validated model form from handbook docs
3. Remove under-used form from comment card
4. Update form for zoom-to-route
5. Update the forms for settings.tokens
6. Remove the `ValidatedModelForm`

## Side Effects

I upgraded the zoom-to-route functionality to allow you to search the routes rather than just scrolling through the whole list each time. I also threw in some error handling for the tokens stuff.

## QA Notes

The main thing to check is if you can create and edit Personal Access Tokens still.


[ENG-4421]: https://openscience.atlassian.net/browse/ENG-4421?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
